CONTRIBUTING.md

```diff
@@ -32,13 +32,10 @@ See [tests/README.md](./tests/README.md) for how to execute and add tests.
 
 ### Updating generated code
 
-DM contains some proto files. To modify them,
-you also need to install:
-
-* [protoc 3.6+](https://github.com/protocolbuffers/protobuf/releases)
```

Comment:
are these two still needed to install manually?

Reply:
i thought it could be installed through `make tools_setup` ,but i was wrong ...
i will restore thie part of doc